rustls/examples/internal/bogo_shim.rs

```diff
@@ -764,7 +781,8 @@ fn exec(opts: &Options, mut sess: Connection, count: usize) {
             let mut one_byte = [0u8];
             let mut cursor = io::Cursor::new(&mut one_byte[..]);
             sess.write_tls(&mut cursor).unwrap();
-            conn.write(&one_byte).expect("IO error");
+            conn.write_all(&one_byte)
```

Comment:
Unrelated?

Reply:
I didn't change this manually. For some reason, running `cargo fmt` keeps forcing it onto a second line on my branch.